### DISCUSSION TITLE
Export wiki as git repo

### DISCUSSION THREAD

**Discussion by iamrubeng** [outline/outline#6790] [Ideas]
Hello,

It could be useful to export wiki as git repo to Azure DevOps or any other instance. 
We use our wiki locally and if a disaster occurs, we could access to wiki using the repo.

Now we do it manually using the API.

**Comment by tommoor** [comment]
This would be cool, yea

**Comment by ReessKennedy** [comment]
@rgomezceis ... you mean you wrote your own code that uses the Outline API to automatically push your Outline wiki content to a Git repo?

**Comment by wolfwithdances** [comment]
Previous conversation on git export as a backup solution: #997, closed in #1013 as `collections.exportAll`.

This isn't quite as direct of a solution as I would have loved to see—I would have preferred something like what captn3m0 pitched in their [Jul 24 2019 comment](https://github.com/outline/outline/issues/997#issuecomment-514728900), particularly if it would mirror the outline edit history via git commits—but it does make it tractable to do an automated single-API-call export with attachments.

**Reply by tommoor** [reply]
I would recommend using webhooks honestly, just listen for `revisions.create` event and write the data to git every time it's received. Export is doable but very heavy handed in comparison
